### PR TITLE
Edit assessment page redesign

### PR DESCRIPTION
**Pull request by nayak16**
- Create materialize tabs
- Redesign module and problems ux
- IMPORTANT: Materialize Datetimepicker does not work yet. Will be done in separate PR
Screenshots

screen shot 2016-12-26 at 6 13 41 pm
screen shot 2016-12-26 at 6 07 29 pm

### CODE REVIEW

**Comment by nayak16** [comment]
@zyx-billy This PR will get you started with styles for a couple other pages. Can you take a look? I'm working on Datetimepicker right now

**Comment by zyx-billy** [comment]
@nayak16 This looks great. I'll start with the extension page then, which also uses forms and tables.

**Comment by remzr7** [comment]
Wouldn't red icons for the trash and edit options look better than the light blue ones?

File: app/views/assessments/_edit_basic.html.erb [hunk 1]
```diff
@@ -1,60 +1,64 @@
...
-    <%= link_to "Delete Autograder", [@course, @assessment, :autograder], method: :delete, class: "btn danger",
-                data: {confirm: "Are you sure you want to delete the Autograder for this assessment?"} %>
-  <% else %>
-    <%= link_to "Add Autograder", [@course, @assessment, :autograder], method: :post, class: "btn primary" %>
-  <% end %>
-</div>
-
-<div class="form-group">
-  <% if @assessment.has_scoreboard? %>
-    <%= link_to "Edit Scoreboard Settings", [:edit, @course, @assessment, :scoreboard], class: "btn primary" %>
```
Comment (remzr7) [inline app/views/assessments/_edit_basic.html.erb]: @nayak16 why don't we have this anymore?

**Comment by nayak16** [comment]
Light Blue implies links. As all the other links on the page are light blue

**Pull request merged**